### DISCUSSION TITLE
Help with new extension PR

### DISCUSSION THREAD

**Discussion by Brackets-Coder** [TurboWarp/extensions#1639] [Q&A] [answered]
I recently submitted a pull request for my small MIDI extension. Initially, it failed both validate and format, but I fixed validate. Since I'm relatively new to GitHub and this was my first extension pull request (I have prior extension development experience), I don't exactly know how to fix format, much less run prettier locally. My question is, does format need to be fixed for the extension to be published and if so how is it fixed?

**Comment by CubesterYT** [comment, marked as answer]
Yes formatting does need to be fixed, but you don't have to do it yourself. Me and GarboMuffin can do the prettier and other fixes for you, once we check it out. If your extension is done, let me know on your PR and I will check it out.

**Reply by Brackets-Coder** [reply]
It's in a finished state that I'm satisfied with enough to put it on the gallery, but it has future potential for some updates. The pull request is submitted, but I can't get prettier done. Thanks for the help!

**Reply by Brackets-Coder** [reply]
I'd like to have my extension published to the gallery, but obviously that can't happen until prettier is fixed. How can I run that locally myself? I'm willing to learn and it'd be better on the long run if I'm able to do prettier on some future extensions.

**Reply by CST1229** [reply]
`npm run format` in the repository's directory (assuming you cloned it and ran `npm ci`), but you don't need to worry about that. A maintainer will fix it before merging anyways.

**Comment by AshimeeAlt** [comment]
it can take from a few hours to over a year
________________________________

**Reply by Brackets-Coder** [reply]
Ok. Good to know. I guess smaller extensions get pushed further down the list whereas bigger extensions like Pen+, AR, and Simple3D are more top priority but take longer to review?

**Comment by AshimeeAlt** [comment]
not really 😕
________________________________

**Reply by Brackets-Coder** [reply]
So are they all kind of the same when it comes to review times?
If these reviews take a while is there anything I can do to help get traffic flowing a little bit better?

**Comment by AshimeeAlt** [comment]
liking the ones you thing should be reviewed and maybe doing some reviews yourself
________________________________

**Reply by Brackets-Coder** [reply]
Ok. I'll get started to liking, but how does reviewing extensions work?

**Comment by AshimeeAlt** [comment]
if you dont know best leave it at just liking for now.
________________________________

**Comment by CubesterYT** [comment]
Some big ones that get merged quicker is because they're made by developers who have already merged extensions and are quite trusted to know what they are doing. So we push them to the finish line quicker without much thought.
